modules/client/src/allowlistVoting/internal/decoding.ts

```diff
@@ -139,30 +142,40 @@ export class AllowlistVotingDecoding {
    * Decoding encoded data or a TX for the setConfiguration function
    *
    * @static
-   * @param {(IDecodingTX | BytesLike)} txOrData
+   * @param {(DecodingTXType | BytesLike)} txOrData
    * @return {*}  {ISetConfigurationParams}
    * @memberof AllowlistVotingDecoding
    */
   public static setConfiguration(
-    txOrData: IDecodingTX | BytesLike
+    txOrData: DecodingTXType | BytesLike
   ): ISetConfigurationParams {
-    return AllowlistVotingDecoding.getDecodedData("setConfiguration", txOrData);
+    const data = AllowlistVotingDecoding.getDecodedData(
+      "setConfiguration",
+      txOrData
+    );
+    return {
+      _minDuration: parseInt(data._minDuration.toString()),
+      _participationRequiredPct: parseInt(
+        data._participationRequiredPct.toString()
+      ),
+      _supportRequiredPct: parseInt(data._supportRequiredPct.toString()),
```

Comment:
Why do all fields have an underscore at the beginning?

Reply:
@josemarinas should there be a precision decoding here?

Reply:
> Why do all fields have an underscore at the beginning?

was also the case for some parameters in the core client. so I guessed that params pushed as function parameters and not as interface are with an `_` to mark them as contract parameters